data/virt_autotest/vm_host_bridge_final.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+BEIDGE_INF=$1
+. /etc/os-release
+ACTIVE_NET=$(ip a|awk -F': ' '/state UP/ {print $2}'|head -n1)
+interface="BOOTPROTO='dhcp'\nSTARTMODE='auto'"
+echo -e $interface >/etc/sysconfig/network/ifcfg-$ACTIVE_NET
+cat /etc/sysconfig/network/ifcfg-$ACTIVE_NET
+rm -rf /etc/sysconfig/network/ifcfg-$BEIDGE_INF*
```

Comment:
LGTM. Just not sure what this $BEIDGE_INF stands for. I am OK if this is the intended spelling.

Reply:
Great, there should be $BRIDGE_INF(i.e. HOST_BRIDGE_INTERFACE) here. 
Had modified $BEIDGE_INF-> $BRIDGE_INF